### PR TITLE
Fix author list parser (#4169)

### PR DESCRIPTION
**Pull request by 1160300208**
- [x] Change in CHANGELOG.md described
- [x] Tests created for changes
- [x] Manually tested changed features in running JabRef
- [x] Screenshots added in PR description (for bigger UI changes)
- [x] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

We have fixed an issue where author list parser can't generate bibtexkey for <B>Chinese author.</B> [#4169](https://github.com/JabRef/jabref/issues/4169)

<h2>Before fix: </h2>

The default bibtexkey is [author][year], actually it only shows [year]. Obviously the [author] can't be generated normally. <br/>
<h2>After fix: </h2>

Now it can generate bibtexkey for <B>Chinese author.</B><br/>
We have used correct logic so that if author's name is not Chinese name, it won't do anything.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Window state is saved on close and restored on start.
 - Files without a defined external file type are now directly opened with the default aplication of the operating system
 - We streamlined the process to rename and move files by removing the confirmation dialogs.
+- We removed the redundant new lines of markings and wrapped the summary in the File annotation tab. [#3823](https://github.com/JabRef/jabref/issues/3823)
```
Comment (tobiasdiez) [inline CHANGELOG.md]: This comment is probably a leftover from the other PR.

Comment (1160300208) [reply CHANGELOG.md]: So what should I do after remove the additional changelog entry? Commit and push?

Comment (1160300208) [reply CHANGELOG.md]: This is my first time pull request.

**Comment by 1160300208** [comment]
It is ok now?

File: CHANGELOG.md (hunk 1 above)
Comment (tobiasdiez) [reply CHANGELOG.md]: Yes, everything is fine now. Thanks! 
We will wait until a second developer gives a positive review before merge.

**Pull request merged**